src/clientSideScene/ClientSideSceneComp.tsx

```diff
@@ -202,12 +202,20 @@ const Overlay = ({
   let xAlignment = overlay.angle < 0 ? '0%' : '-100%'
   let yAlignment = overlay.angle < -90 || overlay.angle >= 90 ? '0%' : '-100%'
 
+  // It's possible for the pathToNode to request a newer AST node
+  // than what's available in the AST at the moment of query.
+  // It eventually settles on being updated.
   const _node1 = getNodeFromPath<Node<CallExpression>>(
     kclManager.ast,
     overlay.pathToNode,
     'CallExpression'
   )
-  if (err(_node1)) return
+
+  // For that reason, to prevent console noise, we do not use err here.
+  if (_node1 instanceof Error) {
```

Comment:
Heh. I think I have to apologize because I'm the one who really pushed err there. After looking deeper, I came to the conclusion it's ok in this very specific instance to not have it there, but decorate it with a comment.

Reply:
I made this: #4482.

Reply:
The thing is though is that this _is_ an error on our side. Luckily this is in an Overlay, which is created and destroyed quite often